hadoop-master1/Dockerfile

```diff
@@ -93,6 +93,7 @@ RUN chown -R zookeeper:hadoop /opt/apache-zookeeper-${ZOOKEEPER_VERSION}-bin &&
     chown -R trino:hadoop /opt/trino-server-${TRINO_VERSION}
 
 RUN chmod +x ${TRINO_HOME}/bin/trino-cli
+RUN chmod 400 /root/.ssh/id_rsa_hadoop_testing
```

Comment:
IIRC the default perm for private key is 600? am I wrong?

Reply:
It seems 400 is enough.

Reply:
prefer to respect the default value.

Reply:
OK